t/20_sdaf_library.t

```diff
@@ -178,6 +252,51 @@ subtest '[serial_console_diag_banner] ' => sub {
     dies_ok { serial_console_diag_banner('exeCuTing deploYment' x 6) } 'Fail with string exceeds max number of characters';
 };
 
+subtest '[sdaf_deploy_workload_zone] Test executed command' => sub {
+    my $ms_sdaf = Test::MockModule->new('sles4sap::sdaf_library', no_auto => 1);
+    my $config_path_command;
+    my $sdaf_command;
+    $ms_sdaf->redefine(assert_script_run => sub { $config_path_command = $_[0] if grep(/^cd\s/, @_); return 0; });
+    $ms_sdaf->redefine(script_run => sub { $sdaf_command = $_[0] if grep(/install_workloadzone.sh/, @_); return 0; });
+    $ms_sdaf->redefine(record_info => sub { return 1; });
+    $ms_sdaf->redefine(set_os_variable => sub { return 1; });
+    $ms_sdaf->redefine(get_os_variable => sub { return '/root/Azure_SAP_Automated_Deployment/WORKSPACES/DEPLOYER/LAB-SECE-DEP05-INFRASTRUCTURE/'; });
+    $ms_sdaf->redefine(upload_logs => sub { return 1; });
+    $ms_sdaf->redefine(log_dir => sub { return '/tmp/openqa_logs'; });
+
+    my $expected_command = '( $SAP_AUTOMATION_REPO_PATH/deploy/scripts/install_workloadzone.sh --parameterfile  --deployer_environment ${deployer_env_code} --deployer_tfstate_key ${deployerState} --keyvault ${key_vault} --storageaccountname ${tfstate_storage_account} --subscription ${ARM_SUBSCRIPTION_ID} --tenant_id ${ARM_TENANT_ID} --spn_id ${ARM_CLIENT_ID} --spn_secret ${ARM_CLIENT_SECRET} --auto-approve 2>&1 | tee /tmp/openqa_logs/deploy_workload_zone.log ; exit ${PIPESTATUS[0]})';
```

Comment:
Other places are composing `deploy/scripts/` path in a different way https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19048/files#diff-a7b59ba45f396aa9b361d43f76fb4f6669a4b42c5d0e1dc952bfa526ec508934R795